tests/ConstantTest.php

```diff
@@ -586,6 +618,8 @@ class B extends A {
 
                     class Clazz {
                         /**
+                         * @var 0|1
```

Comment:
I thought about making it resolve the assignment to `1` instead of `0|1`, but I figure the only reason for a ternary is that the author is planning to change the constant later, in which case I think it might make more sense to force them to use the type annotation. Thoughts?

Reply:
I changed my mind, I think it would be a good idea to infer this, even though it's redundant.